tests/qa/qatests

```diff
@@ -202,17 +207,25 @@ def main(token, test, modver, snapshot, jsonver, rlecver, osnick, light, quick,
     test_fname = os.path.join(HERE, f'{test}.json')
     if not os.path.exists(test_fname):
         raise click.ClickException(f"Invalid test name: {test}")
+    if light and vecsim:
+        raise click.ClickException("Invalid combination: light and vecsim")
```

Comment:
why?

Reply:
right. no conflict there in principle, however, when defining `SEARCH_TEST_NAME` it's either `RedisearchLight` or `RediSearchVecsim` (due to the way its built in Opereto tests), so hence the conflict in practice.